shuffle/README.md

```diff
@@ -103,3 +103,22 @@ newly created account, (3) publish another `SampleModule::Coin` resource to the
 
 The first two transactions should succeed and the last one should fail because you can only publish
 one resource of each type under an address.
+
+
+## Step 4: Publish Modules 
+
+Write any module, like SampleUserModule.move in `/move/src/modules/user_modules` and put in in the user_modules directory.
```

Comment:
nit: can we rename `user_modules` to not have two words, esp with a second modules being redundant since it's a child of modules/. One suggestion: `/move/src/modules/user`. Or we still guiding them down the notion of a package folder.